Gemfile

```diff
@@ -2,8 +2,7 @@ source 'https://rubygems.org'
 git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
 # ruby '3.1.2'
-ruby '3.0.0'
-# ruby '3.2.0'
+ruby '3.2.0'
```

Comment:
@TracyMuso  can you try always to remove redundant code from your code as it makes it look clean